README.md

```diff
@@ -105,6 +97,25 @@ cargo build --workspace
 cargo test --workspace
 ```
 
+#### Build on MacOS with Docker VirtuoFS
+
+As reported in [this issue](https://github.com/eclipse-chariott/chariott/issues/111), some extra
+steps are needed to build if you are using MacOS with Docker VirtuoFS.
+
+Docker Desktop provides a fast VirtuoFS implementation on a Mac but rust build process breaks in
+devcontainers if VirtuoFS is enabled. The solution is not disabling the VirtuoFS, because this
+significantly slows down the I/O operations in the containers.
+
+The workaround is to create a target folder outside the chariott workspace and set the environment
+variable to the new target folder. The following sequence works in the devcontainer:
+
+```shell
+vscode ➜ /workspaces/chariott (main) $ sudo mkdir ../target
+vscode ➜ /workspaces/chariott (main) $ sudo chown vscode:vscode ../target
+vscode ➜ /workspaces/chariott (main) $ CARGO_TARGET_DIR=../target cargo build -p cargo
```

Comment:
I think that these 2 lines should be:

export CARGO_TARGET_DIR=../target
cargo build -p cargo
cargo run -p chariott

Reply:
You may be able to drop the export when it runs in vscode.

Reply:
As discussed, this is a valid way of specifying the cargo target directory (https://stackoverflow.com/questions/50364390/how-can-i-specify-a-custom-cargo-output-directory). Given that this was a fix suggested by another contributor and we've validated that these commands work, I will leave it as-is.